### DISCUSSION TITLE
Markdown_V2 isn't working for update.message.reply_text

### DISCUSSION THREAD

**Discussion by cal-cmd** [python-telegram-bot/python-telegram-bot#4330] [Q&A]
Have this code below:

`await update.message.reply_text(await main_menu_message(), reply_markup=await main_menu_keyboard(), parse_mode=telegram.constants.ParseMode.MARKDOWN_V2)`

and have imported everything - here is the error I get back:

`await update.message.reply_text(await main_menu_message(), reply_markup=await main_menu_keyboard(), parse_mode=telegram.constants.ParseMode.MARKDOWN_V2)
NameError: name 'telegram' is not defined
`

**Comment by Bibo-Joshi** [comment]
https://stackoverflow.com/q/78690707

Please don't ask the same question in multiple places at once. 

NameError: name 'telegram' is not defined

This means that you have in fact not imported `telegram` ;)

**Comment by cal-cmd** [comment]
Ahhh my bad! And yes, was very confused by the explanation but finally
figured it out thanks to the kind folks. This should be closed already (:
